frontend/talentsearch/src/js/components/reviewMyApplication/ReviewMyApplicationPage.tsx

```diff
@@ -0,0 +1,266 @@
+import * as React from "react";
+import { useIntl } from "react-intl";
+import UserProfile from "@common/components/UserProfile";
+import Pending from "@common/components/Pending";
+import NotFound from "@common/components/NotFound";
+import { commonMessages } from "@common/messages";
+import { getLocale } from "@common/helpers/localize";
+import MissingSkills from "@common/components/skills/MissingSkills";
+import { notEmpty } from "@common/helpers/util";
+import ExperienceSection from "@common/components/UserProfile/ExperienceSection";
+import { Link } from "@common/components";
+import { getMissingSkills } from "@common/components/skills/MissingSkills/MissingSkills";
+import {
+  Applicant,
+  Maybe,
+  PoolAdvertisement,
+  useGetReviewMyApplicationPageDataQuery,
+} from "../../api/generated";
+import { useDirectIntakeRoutes } from "../../directIntakeRoutes";
+import ApplicationPageWrapper from "../ApplicationPageWrapper/ApplicationPageWrapper";
+import { flattenExperienceSkills } from "../experienceAndSkills/ExperienceAndSkills";
+
+interface ReviewMyApplicationProps {
+  applicant: Applicant;
+  poolAdvertisement: PoolAdvertisement;
+  poolCandidateId: string;
+  closingDate: Date;
+  jobTitle: Maybe<string>;
+  poolId: string;
+}
+
+export const ReviewMyApplication: React.FunctionComponent<
```

Comment:
Could you add a Storybook story for this page?

Reply:
Fixed by [Add storybook for ReviewMyApplication](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3796/commits/32c262e1665c982651ee74ba666313d91cf6da20)